server/index.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const db = require('../database');
+
+const app = express();
+const PORT = process.env.PORT || 3000;
+// TODO - implement solution for backend
+
+// Create GET API endpoint
+// TODO - Create all database connections
+// TODO -
+
+// Serve the public folder for client
+app.use(express.static(__dirname + '/../public'));
```

Comment:
__dirname works here? I couldn't get it to work for me...

Reply:
it certainly worked for me...